Dockerfile-local

```
@@ -1,4 +1,4 @@
-FROM golang:1.24.1-bullseye as build
+FROM golang:1.24.6-bookworm as build
```

Local dockerfile does not need to be bookworm, only CI ones.

```suggestion
FROM golang:1.24.6 as build
```